src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts

```diff
@@ -184,16 +199,63 @@ class F extends GPUTest {
     // Verify the content of the whole subresouce of dstTexture at dstCopyLevel (in dstBuffer) is expected.
     this.expectContents(dstBuffer, expectedUint8DataWithPadding);
   }
+
+  static kCopyBoxOffsetsForWholeDepth = [
+    // From (0, 0) of src to (0, 0) of dst.
+    {
+      srcOffset: { x: 0, y: 0, z: 0 },
+      dstOffset: { x: 0, y: 0, z: 0 },
+      copyExtent: { width: 0, height: 0, depth: 0 },
+    },
+    // From (0, 0) of src to (1, 0) of dst.
+    {
+      srcOffset: { x: 0, y: 0, z: 0 },
+      dstOffset: { x: 1, y: 0, z: 0 },
+      copyExtent: { width: 0, height: 0, depth: 0 },
+    },
+    // From (0, 0) of src to (0, 1) of dst.
+    {
+      srcOffset: { x: 0, y: 0, z: 0 },
+      dstOffset: { x: 0, y: 1, z: 0 },
+      copyExtent: { width: 0, height: 0, depth: 0 },
+    },
+    // From (1, 0) of src to (0, 0) of dst.
+    {
+      srcOffset: { x: 1, y: 0, z: 0 },
+      dstOffset: { x: 0, y: 0, z: 0 },
+      copyExtent: { width: 0, height: 0, depth: 0 },
+    },
+    // From (0, 1) of src to (0, 0) of dst.
+    {
+      srcOffset: { x: 0, y: 1, z: 0 },
+      dstOffset: { x: 0, y: 0, z: 0 },
+      copyExtent: { width: 0, height: 0, depth: 0 },
+    },
+    // From (1, 0) of src to (0, 0) of dst, and the copy extent will not cover the last
+    // column of both source and destination texture.
+    {
+      srcOffset: { x: 1, y: 0, z: 0 },
+      dstOffset: { x: 0, y: 0, z: 0 },
+      copyExtent: { width: -1, height: 0, depth: 0 },
+    },
+    // From (0, 1) of src to (0, 0) of dst, and the copy extent will not cover the last
+    // row of both source and destination texture.
+    {
+      srcOffset: { x: 0, y: 1, z: 0 },
+      dstOffset: { x: 0, y: 0, z: 0 },
+      copyExtent: { width: 0, height: -1, depth: 0 },
+    },
+  ];
 }
 
 export const g = makeTestGroup(F);
 
-// TODO(jiawei.shao@intel.com): support all WebGPU texture formats
-g.test('t2t_non_array_non_compressed_color_formats')
+g.test('color_textures_non_compressed_non_array')
```

Comment:
Name should use commas to separate parts of the test "path" (like slashes in a file path, but we use comma, at least right now) e.g.
```suggestion
g.test('color_textures,non_compressed,non_array')
```